extensions/notebook/src/book/bookTreeItem.ts

```diff
@@ -13,7 +13,6 @@ export class BookTreeItem extends vscode.TreeItem {
 		public readonly tableOfContents: any[],
 		public readonly collapsibleState: vscode.TreeItemCollapsibleState,
 		public uri?: string,
-		public readonly type?: vscode.FileType,
```

Comment:
can you explain why this was removed?

Reply:
I realized that we don't need to use this anywhere. I had originally copied from an example online. Sorry, I should've explained this in the PR.